crates/matrix-sdk-sled/src/state_store.rs

```diff
@@ -107,7 +107,7 @@ impl Into<StoreError> for SledStoreError {
                 )),
             },
             SledStoreError::StoreError(e) => e,
-            _ => StoreError::Backend(anyhow!(self)),
+            _ => StoreError::Backend(Box::new(self)),
```

Comment:
I think we might have a similar thing in the indexeddb-store ... let's see what CI has to say about it :)

Reply:
Right, if `.into()` doesn't work, this might.